deploy/docker-compose.yaml

```diff
@@ -2,33 +2,44 @@ version: '3'
 
 services:
   alphafold:
-    image: nvcr.io/nim/deepmind/alphafold2:1.0.0
+    image: nvcr.io/nim/deepmind/alphafold2:2.0.0
```

Comment:
Recommend bumping version to 2.1.0 and retesting. This should fix the incorrect org / team variable bug.

Reply:
Easy fix.

Reply:
Bumping version and retesting

Reply:
working, fixed in most recent commit